src/Factories/FunFunFactory.php

```diff
@@ -282,44 +282,52 @@ private function newPimple(): Container {
 		$pimple['translator'] = function() {
 			$translationFactory = new TranslationFactory();
 			$loaders = [
-				'json' => $translationFactory->newJsonLoader()
+				'json' => $translationFactory->newJsonLoader(),
+				'directory' => $translationFactory->newDirectoryLoader()
 			];
 			$locale = $this->config['locale'];
 			$translator = $translationFactory->create( $loaders, $locale );
 			$translator->addResource(
 				'json',
-				__DIR__ . '/../../app/fundraising-frontend-content/i18n/' . $locale . '/messages.json',
+				__DIR__ . '/../../' . $this->config['i18n-base-path'] . '/' . $locale . '/messages.json',
```

Comment:
I'd pull this concatenation out into a variable and use it for all `addResource` calls.